docs.feldera.com/docs/pipelines/checkpoint-sync.md

```diff
@@ -34,14 +35,23 @@ Here is a sample configuration:
-| `access_key`              | `string`  | Your S3 access key. Not required if using environment-based authentication (e.g., IRSA).                                                      |
-| `secret_key`              | `string`  | Your S3 secret key. Same rules as `access_key`.                                                                                               |
-| `start_from_checkpoint`   | `string`  | Provide a checkpoint UUID to resume from it, or use `latest` to restore from the latest one. The provided UUID must exist in object store.    |
+| Field                   | Type            | Description                                                                                                                                                                          |
+|-------------------------|-----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `endpoint`              | `string`        | The S3-compatible object store endpoint (e.g., `http://localhost:9000` for MinIO).                                                                                                   |
+| `bucket` \*             | `string`        | The bucket name and optional prefix to store checkpoints (e.g., `mybucket/checkpoints`).                                                                                             |
+| `region`                | `string`        | The region of the bucket. Leave empty for MinIO or use AWS default like `us-east-1`.                                                                                                 |
+| `provider` \*           | `string`        | The S3 provider identifier. Must match [rclone’s list](https://rclone.org/s3/#providers). Case-sensitive. Use `"Other"` if unsure.                                                   |
+| `access_key`            | `string`        | Your S3 access key. Not required if using environment-based auth (e.g.,  IRSA).                                                                                                      |
+| `secret_key`            | `string`        | Your S3 secret key. Not required if using environment-based auth.                                                                                                                    |
+| `start_from_checkpoint` | `string`        | Provide a checkpoint UUID to resume from it, or use `latest` to restore from the latest one. The provided UUID must exist in object store.                                           |
+| `transfers`             | `integer (u8)`  | Number of concurrent file transfers. Default: `20`.                                                                                                                                  |
```

Comment:
In other connector docs, I have a separate column for the default value. Can we do the same here?